Makefile

```diff
@@ -35,9 +35,12 @@ full-help: $(VENVDIR)
 	@. $(VENV); $(SPHINXBUILD) -M help "$(SOURCEDIR)" "$(BUILDDIR)" $(SPHINXOPTS) $(O)
 	@echo "\n\033[1;31mNOTE: This help texts shows unsupported targets!\033[0m"
 	@echo "Run 'make help' to see supported targets."
-	
+
+# Must not assume the venv module is available because Reasons (capital R):
```

Comment:
I don't understand this comment ... what do you mean with "Reasons (capital R)"?
If it's some kind of reference, this is maybe not the place for it. ;-)